bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/BookieInitializationTest.java

```diff
@@ -1576,4 +1576,17 @@ private void bookieConnectAfterCookieDeleteWorker(ServerConfiguration conf, Regi
             LOG.info("As expected Bookie fails to come up without a cookie in metadata store.");
         }
     }
+
+    @Test
+    public void testInvalidServiceMetadataURI() throws Exception {
+        ServerConfiguration conf = TestBKConfiguration.newServerConfiguration();
+        conf.setMetadataServiceUri("zk+flat:///ledgers"); // default value in docker image
```

Comment:
a more sensible example is zk+null and not zk+flat

we could also add tests about other silly cases, like:
- single slash
- no slash at all
- empty string
- null string